README.md

```diff
@@ -104,6 +104,10 @@ To check the status of your IPsec VPN server, you can pass `ipsec status` to you
 docker exec -it l2tp-ipsec-vpn-server ipsec status
 ```
 
+### Scaleway user information
```

Comment:
Hi,
Scaleway seems to be a brand, so can you add the copyright symbol after Scaleway string please.
Example : Scaleway ©.

Reply:
Ok.
* I added copyrate mark
* I added link to Scaleway home page